src/duckstation-qt/debuggerwindow.cpp

```diff
@@ -251,10 +251,10 @@ void DebuggerWindow::onCodeViewContextMenuRequested(const QPoint& pt)
   menu.addAction(QStringLiteral("0x%1").arg(static_cast<uint>(address), 8, 16, QChar('0')))->setEnabled(false);
   menu.addSeparator();
 
-  QAction* action = menu.addAction(QIcon(":/icons/media-record@2x.png"), tr("Toggle &Breakpoint"));
+  QAction* action = menu.addAction(tr("Toggle &Breakpoint"));
   connect(action, &QAction::triggered, this, [this, address]() { toggleBreakpoint(address); });
 
-  action = menu.addAction(QIcon(":/icons/debug-run-cursor.png"), tr("&Run To Cursor"));
+  action = menu.addAction(tr("&Run To Cursor"));
```

Comment:
Was removing the icons here intentional? You can use QIcon::fromTheme to create a light/dark icon.

Reply:
Whoops, i thought we don't need it since the theme is already defined on the .ui file

Reply:
This one's being created in the CPP file, not ui

Reply:
Just noticed that its for the context menu icon, fixed:

![image](https://github.com/stenzek/duckstation/assets/14798312/a04103ea-afe6-4f18-874b-6eb0157c9591)